Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.SqlConsole/App_LocalResources/sqlconsole.nl-NL.resx

```diff
@@ -12,88 +12,88 @@
   </data>
-  <data name="ExportExcel.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>Exporteren naar Excel</value>
+  <data name="ExportExcel.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">Exporteren naar Excel</value>
   </data>
-  <data name="ExportPDF.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>Exporteren naar PDF</value>
+  <data name="ExportPDF.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">Exporteren naar PDF</value>
   </data>
-  <data name="nav_SqlConsole.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>SQL Console</value>
+  <data name="nav_SqlConsole.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">SQL Console</value>
   </data>
-  <data name="NewQuery.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>&lt;Nieuwe query&gt;</value>
+  <data name="NewQuery.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">&lt;Nieuwe query&gt;</value>
   </data>
-  <data name="NoData.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>De query heeft geen data teruggegeven.</value>
+  <data name="NoData.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">De query heeft geen data teruggegeven.</value>
   </data>
-  <data name="PageInfo.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>Toont {0} - {1} van {2} items</value>
+  <data name="PageInfo.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">Toont {0} - {1} van {2} items</value>
   </data>
-  <data name="PageSize.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>{0} items</value>
+  <data name="PageSize.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">{0} items</value>
   </data>
-  <data name="Query.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>Query:</value>
+  <data name="Query.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">Query:</value>
   </data>
-  <data name="QueryFailed.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>De query niet succesvol!</value>
+  <data name="QueryFailed.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">De query niet succesvol!</value>
   </data>
-  <data name="QuerySuccessful.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>De query is succesvol voltooid!</value>
+  <data name="QuerySuccessful.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">De query is succesvol voltooid!</value>
   </data>
-  <data name="QueryTabTitle.Text" xml:space="preserve" lastModified="2016-12-01 14:29:16">
-    <value>Query {0}</value>
+  <data name="QueryTabTitle.Text" lastModified="2016-12-01 13:29:16">
+    <value xml:space="preserve">Query {0}</value>
```

Comment:
is xml:space="preserve" valid on the `value` node, I always saw that in the `data` node? Was this manually moved, or what tool was used and was it tested in Dnn to confirm it is read correctly?

Reply:
I did not change that, this is what the pack was/is.